### PR TITLE
Shorten @foo//:foo labels to @foo

### PR DESCRIPTION
**Pull request by jart**
This convenient syntax been available in Bazel since November. See https://github.com/bazelbuild/bazel/commit/61affe77f9f5e5e7fcbbba7d8c2bcf8fbab776d5

Attn: @pmbethe09

### CODE REVIEW

File: build/rewrite.go [hunk 1]
```diff
@@ -194,9 +195,9 @@ func fixLabels(f *File, info *RewriteInfo) {
 		*p = str1
 	}
 
-	// labelRE matches label strings //x/y/z:abc.
-	// $1 is //x/y/z, $2 is x/y/, $3 is z, $4 is :abc, and $5 is abc.
-	labelRE := regexp.MustCompile(`^(//(.*/)?([^:]+))(:([^:]+))?$`)
+	// labelRE matches label strings, e.g. @r//x/y/z:abc
+	// where $1 is @r//x/y/z, $2 is r, $3 is z, $4 is abc.
+	labelRE := regexp.MustCompile(`^((?:@(\w+))?//(?:.*/)?([^:]*))(?::([^:]+))?$`)
```
Comment (pmbethe09) [inline build/rewrite.go]: Can you move this labelRE to a file level var?  
previous bug: MustCompile should only be used in vars...

**Comment by bolinfest** [comment]
Does this honor `-buildifier_disable=label`?

File: build/rewrite.go (hunk 1 above)
Comment (jart) [reply build/rewrite.go]: Done.

**Comment by laurentlb** [comment]
Thanks, LGTM

**Pull request merged**